vic/vic_run/src/solve_snow.c

```diff
@@ -354,7 +354,7 @@ solve_snow(char               overstory,
             energy->AlbedoUnder = *AlbedoUnder;
 
             /** Compute Snow Parameters **/
-            if (snow->swq > 0.) {
+            if (snow->swq > DBL_EPSILON) {
```

Comment:
I'm not sure, I should look into this more.  We were ending up in this section when `snow->swq == 0` and were dividing by zero so this was originally added when I was debugging.  I'll try removing it again and we'll see if it is still needed.